__tests__/utilities/test-utils.js

```diff
@@ -0,0 +1,80 @@
+const fs = require('fs');
+const _ = require('lodash');
+const path = require("path");
+const RudderCDK = require("rudder-transformer-cdk");
+
+function formTestParams(dest, transformAt) {
+	const version = "v0";
+  
+  const transformer = require(
+    path.resolve(__dirname + `../../../${version}/destinations/${dest}/transform`)
+  );
```

Comment:
Why are we expecting `transform.js` to be always present?

Reply:
Because that is how we implement a destination & it is how we're even getting a handler in `versionedRouter.js`
Please take a look at https://github.com/rudderlabs/rudder-transformer/blob/master/versionedRouter.js#L42-L47

Reply:
Why do we expect it to be present for CDK-type destinations?

Reply:
We are just requiring it in a more like a higher context but may be we can only require it for the destinations other than cdk

Reply:
Even for CDK, you can have `transform.js`, right? which will host extraneous functions referenced in the config. The concern is that no error handling is done.